docs/man/skstat.8.adoc

```diff
@@ -315,8 +315,8 @@ The security or encryption method, if any, used for this connection.
 authentication::
 The authentication method and user ID of the connection's authenticated user.
 
-tenant::
-If the connection is to a listener using multi-tenancy, this column displays the tenant namespace for the connection.
+active::
+Whether this edge connection is active or standby.
```

Comment:
how do i get this in output:

```
kubectl exec --stdin --tty deploy/skupper-router -- skstat -e

Defaulted container "router" out of: router, config-sync
2023-01-11 11:45:40.052263 UTC
east-skupper-router-6c8866fc67-nc448
```

Reply:
To see any of the columns, you would have to have at least one edge router connected to the router you are querying.